pkg/util/pod/pods.go

```diff
@@ -7,20 +7,38 @@ import (
 	"github.com/maistra/maistra-test-tool/pkg/util/test"
 )
 
+func getPods(t test.TestHelper, oc *ocpackage.OC, selector string, ns string) []ocpackage.NamespacedName {
+	t.T().Helper()
+	output := oc.Invokef(t, "kubectl -n %s get pods -l %q -o jsonpath='{.items[*].metadata.name}'", ns, selector)
+	if output == "" {
+		t.Fatalf("no pods found using selector %s in namespace %s", selector, ns)
+	}
+	pods := strings.Split(output, " ")
+	var namespacedNames []ocpackage.NamespacedName
```

Comment:
After you inline `pods`, you can then rename `namespacedNames` to `pods`.

Reply:
OK